### PR TITLE
feat: Add ConnectionStringParser

### PR DESCRIPTION
**Pull request by markwolff**
Related to https://github.com/microsoft/ApplicationInsights-node.js/pull/538

### CODE REVIEW

File: AISKU/src/Initialization.ts [hunk 1]
```diff
@@ -50,6 +50,13 @@ export class Initialization implements IApplicationInsights {
...
 
+        if (config.connectionString) {
+            const cs = ConnectionStringParser.parse(config.connectionString);
+            const ingest = cs.ingestionendpoint;
+            config.endpointUrl = ingest ? `${ingest}/v2/track` : config.endpointUrl; // only add /v2/track when from connectionstring
+            config.instrumentationKey = cs.instrumentationkey || config.instrumentationKey;
+        }
+
         // ensure instrumentationKey is specified
         if (config && !config.instrumentationKey) {
```
Comment (hectorhdzg) [inline AISKU/src/Initialization.ts]: You can specify connectionString instead of instrumentationKey, here it will override the config set up previously

**Comment by TimothyMothra** [comment]
hi Mark, as part of this PR could you update the Readme with an example snippet of how to set the connection string?

**Comment by markwolff** [comment]
> hi Mark, as part of this PR could you update the Readme with an example snippet of how to set the connection string?

:+1: done e4ffd10

**Pull request merged**